apicast/src/configuration_loader.lua

```diff
@@ -128,7 +128,11 @@ end
 function boot.init_worker(configuration)
   local interval = ttl() or 0
 
-  configuration.keycloak = keycloak.load_configuration()
+  local keycloak_config = _M.run_external_command("keycloak")
```

Comment:
We should move this to the `.init` function. `init_worker` is executed for each worker, but `init` just once.